labs/lab4-shardedstore/tst/dslabs/shardkv/ShardStoreVizConfig.java

```diff
@@ -43,26 +45,48 @@ public SearchState getInitialState(String[] args) {
         int numServersPerGroup = Integer.parseInt(args[1]);
         int numShardMasters = Integer.parseInt(args[2]);
         int numClients = Integer.parseInt(args[3]);
-        List<String> commands = commands(args[4]);
+        int commandStart = 4;
+        int commandEnd = args.length;
         List<String> configCommands = null;
-        if (args.length > 5) {
-            configCommands = commands(args[5]);
+        if (args[args.length - 1].startsWith("config=")) {
```

Comment:
I know it's a bit more annoying to do, but since the string formats of KV commands and ShardMaster commands don't conflict, it would be nice to just detect if the last workload is a KV or ShardMaster workload that way. No need for the `config=`.

Reply:
You could probably detect KV vs. ShardMaster commands with a `try`/`catch`? That should be fairly easy and would be less fragile.

Reply:
Fixed in [8d208c8](https://github.com/emichael/dslabs/pull/19/commits/8d208c8eac09dae0e84b9cac7329fce933f5a726) (and tested it with the commands listed at the bottom of the commit message of [875ff66](https://github.com/emichael/dslabs/pull/19/commits/875ff66a33fd7468b6d73694efbf07c3aea65c53), but without the "config=" part).